packages/fxa-auth-server/lib/oauth/logging/index.ts

```diff
@@ -1,13 +1,12 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Logger } from 'mozlog';
```

Comment:
Import here can properly bring in the type, while the require came in as a value.